internal/db/schema/migrations/postgres/2/02_vault_credential.up.sql

```diff
@@ -55,13 +55,36 @@ begin;
   create trigger delete_credential_store_subtype after delete on credential_vault_store
     for each row execute procedure delete_credential_store_subtype();
 
+  create table credential_vault_token_status_enm (
+    name text primary key
+      constraint only_predefined_token_statuses_allowed
+      check (
+        name in (
+          'current',
+          'maintaining',
+          'revoked',
+          'expired'
+        )
+      )
+  );
+  comment on table credential_vault_token_status_enm is
+    'credential_vault_token_status_enm is an enumeration table for the status of vault tokens.'
```

Comment:
nit: I noticed in your other multi line comments you add a space at the end, I believe you mentioned it concats the lines